*nix package scripts does not work on WSL

**Describe the bug**
package-webapi.sh and other package scripts fail on dotnet publish command.

**To Reproduce**
Steps to reproduce the behavior:
1. change to scripts/deploy directory 
2. run ./package-webapi.sh


**Expected behavior**
Successfully built and published package.

**Error message**
./package-webapi.sh
Building backend executables...
Build configuration: Release
MSBuild version 17.7.3+8ec440e68 for .NET
MSBUILD : error MSB1008: Only one project can be specified.
    Full command line: '/usr/share/dotnet/sdk/7.0.402/MSBuild.dll -maxcpucount -verbosity:m -target:Restore --property:_IsPublishing=true -property:PublishDir=/home/user/repo/chat-compilot/scripts/deploy/publish -property:_CommandLineDefinedOutputPath=true -property:SelfContained=True -property:_CommandLineDefinedSelfContained=true -property:RuntimeIdentifier=win-x64 -property:_CommandLineDefinedRuntimeIdentifier=true -property:Configuration=Release /home/user/repo/chat-compilot/scripts/deploy/../../webapi/CopilotChatWebApi.csproj //p:AssemblyVersion=0.0.0 //p:FileVersion=0.0.0 //p:InformationalVersion= -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,/usr/share/dotnet/sdk/7.0.402/dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,/usr/share/dotnet/sdk/7.0.402/dotnet.dll'
  Switches appended by response files:
Switch: //p:AssemblyVersion=0.0.0

For switch syntax, type "MSBuild -help"

**Platform**
 - OS: Win10, WSL.
 - Language: bash
 - Source: https://github.com/microsoft/chat-copilot/blob/main/scripts/deploy/package-webapi.sh

**Additional context**
not sure why //p: argument is used. If I change it to -p: it works as expected.

--- a/scripts/deploy/package-memorypipeline.sh
+++ b/scripts/deploy/package-memorypipeline.sh
@@ -92,9 +92,9 @@ dotnet publish "$SCRIPT_ROOT/../../memorypipeline/CopilotChatMemoryPipeline.cspr
     --runtime $RUNTIME \
     --self-contained \
     --output "$PUBLISH_OUTPUT_DIRECTORY" \
-    //p:AssemblyVersion=$VERSION \
-    //p:FileVersion=$VERSION \
-    //p:InformationalVersion=$INFO
+    -p:AssemblyVersion=$VERSION \
+    -p:FileVersion=$VERSION \
+    -p:InformationalVersion=$INFO
 
 if [ $? -ne 0 ]; then
     exit 1

--- a/scripts/deploy/package-plugins.sh
+++ b/scripts/deploy/package-plugins.sh
@@ -95,9 +95,9 @@ for PLUGIN_PROJECT_FILE in $PLUGIN_PROJECT_FILES; do
         --configuration $CONFIGURATION \
         --framework $DOTNET \
         --output "$PUBLISH_OUTPUT_DIRECTORY" \
-        //p:AssemblyVersion=$VERSION \
-        //p:FileVersion=$VERSION \
-        //p:InformationalVersion=$INFO
+        -p:AssemblyVersion=$VERSION \
+        -p:FileVersion=$VERSION \
+        -p:InformationalVersion=$INFO
 
     if [ $? -ne 0 ]; then
         exit 1

--- a/scripts/deploy/package-webapi.sh
+++ b/scripts/deploy/package-webapi.sh
@@ -99,9 +99,9 @@ dotnet publish "$SCRIPT_ROOT/../../webapi/CopilotChatWebApi.csproj" \
     --runtime $RUNTIME \
     --self-contained \
     --output "$PUBLISH_OUTPUT_DIRECTORY" \
-    //p:AssemblyVersion=$VERSION \
-    //p:FileVersion=$VERSION \
-    //p:InformationalVersion=$INFO
+    -p:AssemblyVersion=$VERSION \
+    -p:FileVersion=$VERSION \
+    -p:InformationalVersion=$INFO
 
 if [ $? -ne 0 ]; then
     exit 1

